test/csuite/incr_backup/main.c

```diff
@@ -393,7 +393,7 @@ table_changes(WT_SESSION *session, TABLE *table)
                 testutil_check(cur->update(cur));
                 break;
             case _OPERATION_TYPE_COUNT:
-                break;
+                testutil_die(0, "Unexpected OPERATION_TYPE: _OPERATION_TYPE_COUNT");
```

Comment:
My bad, we can probably keep it like this then !

Reply:
What is the Coverity error actually? The ticket doesn't say. Maybe instead of a case for `_OPERATION_TYPE_COUNT` we just have a default case? I don't know if that would satisfy coverity.

Reply:
Thanks for noticing that there's no error in the ticket description. I've updated the ticket. The error is:
_dead_error_condition: The switch governing value op_type cannot be _OPERATION_TYPE_COUNT.
CID 114074 (#1 of 1): Logically dead code (DEADCODE)dead_error_begin: Execution cannot reach this statement: case _OPERATION_TYPE_COUNT:._
 
I tried to just add a default case when I first started the ticket. But compilation failed complaining that not all enum elements were handled in the switch block. Then I tried to fall through _OPERATION_TYPE_COUNT case like that:

case _OPERATION_TYPE_COUNT:
default:
....

It complained that default block handles all the unhandled enums...